packages/launchpad/cypress/e2e/scaffold-component-testing.cy.ts

```diff
@@ -112,6 +112,11 @@ describe('scaffolding component testing', {
       cy.findByTestId('dependency-react-dom').within(() => {
         cy.get('[aria-label="installed"]').should('exist')
       })
+
+      // now clean up the state that we mutated
+      cy.withCtx(async (ctx) => {
```

Comment:
not a problem in CI but when rerunning this spec locally it fails due to polluted state in the project under test's dependencies